src/utils/googleAnalytics.ts

```diff
@@ -8,6 +8,8 @@ import { loadFromCookie, removeCookie } from 'src/logic/cookies/utils'
 
 export const SAFE_NAVIGATION_EVENT = 'Safe Navigation'
 
+// TODO: Update cookie handling for unified app
```

Comment:
Is this TODO still pending?

Reply:
This is covered by [another issue](https://github.com/gnosis/safe-react/issues/2703) in our current sprint.

Reply:
perfect! so should we remove the comment?

Reply:
I'll remove it

Reply:
When the other issue is completed, sure